Doc/library/sqlite3.rst

```diff
@@ -1609,7 +1604,9 @@ Row objects
    It supports iteration, equality testing, :func:`len`,
    and :term:`mapping` access by column name and index.
 
-   Two row objects compare equal if have equal columns and equal members.
+   Two row objects compare equal if they have equal columns and equal members.
```

Comment:
```suggestion
   Two :class:`!Row` objects compare equal
   if they have equal columns and equal members.
```

We really should clarify this is referring to objects of the `Row` class and use the appropriate markup, rather than just arbitrary row-like objects.